Frontend/src/App.js

```diff
@@ -1,26 +1,43 @@
-import logo from './logo.svg';
+import { useRef, useState } from 'react';
 import './App.css';
 import Canvas from './components/Canvas';
+import Constants from './constants/constants';
 
 function App() {
+  const constants = new Constants();
+  const canvasRef = useRef(null);
+  const [LAPI_resp, setLAPI_Resp] = useState(null);
+  const getLatex = async () => {
```

Comment:
this functions should be in separate module as it's not the part of App module by structure. Rest looks good.